subsys/net/l2/openthread/openthread.c

```diff
@@ -397,9 +405,11 @@ int openthread_stop(struct openthread_context *ot_context)
 
 	openthread_api_mutex_lock(ot_context);
 
-	error = otThreadSetEnabled(ot_context->instance, false);
-	if (error == OT_ERROR_INVALID_STATE) {
-		NET_DBG("Openthread interface was not up [%d]", error);
+	if (!IS_ENABLED(CONFIG_OPENTHREAD_COPROCESSOR_RCP)) {
```

Comment:
Let's exit early with 
```
	if (IS_ENABLED(CONFIG_OPENTHREAD_COPROCESSOR)) {
		return 0;
	}
```
instead of modyfing the existing code, no build errors.

Reply:
Done.